examples/knowledge-base/pages/api/search.ts

```diff
@@ -38,7 +38,7 @@ const truncateContent = async (found: lunr.Index.Result) => {
   };
 };
 
-const searchIndex = path.resolve(process.cwd(), 'static', "searchIndex.json");
+const jsonPath = path.join(process.cwd(), 'json');
```

Comment:
Vercel will only let you write to `os.tmpdir()`. Another and perhaps better possibility is writing to `/public` in just the prebuild script and then removing it from the API handler.